src/jit-analyze/Program.cs

```diff
@@ -249,11 +249,24 @@ public static List<FileInfo> ExtractFileInfo(string path, string filter, string
             {
                 string fileNamePattern = filter ?? "*";
                 string searchPattern = fileNamePattern + fileExtension;
+                if (Get(_command.ConcatFiles))
+                {
+                    return new List<FileInfo>
+                    {
+                        new FileInfo
+                        {
+                            name = Path.GetFileName(path),
+                            methodList = ExtractMethodInfo(Directory.EnumerateFiles(fullRootPath, searchPattern, searchOption).ToArray()),
+                            isExplicitOnlyFile = true,
+                        },
+                    };
+                }
+
                 return Directory.EnumerateFiles(fullRootPath, searchPattern, searchOption)
                          .AsParallel().Select(p => new FileInfo
                          {
                              name = p.Substring(fullRootPath.Length).TrimStart(Path.DirectorySeparatorChar),
-                             methodList = ExtractMethodInfo(p)
+                             methodList = ExtractMethodInfo(new[] { p })
```

Comment:
nit: Are we not on a new enough version such that we can use collection initializers instead?
```suggestion
                             methodList = ExtractMethodInfo([p])
```

Reply:
Not yet, jitutils still targets .NET 6.